internal/pkg/service/stream/mapping/table/column/template_benchmark_test.go

```diff
@@ -0,0 +1,22 @@
+package column_test
+
+import (
+	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/api/receive/receivectx"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/mapping/table/column"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"testing"
+)
+
+func BenchmarkColumn_Template_Jsonnet(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		c := column.Template{Language: column.TemplateLanguageJsonnet, Content: "Body('key1[1].key3')"}
+
+		body := `{"key1":[{"key2":"val2"},{"key3":"val3"}]}`
+		header := http.Header{"Content-Type": []string{"application/json"}}
+
+		val, err := c.CSVValue(&receivectx.Context{Body: body, Headers: header})
+		assert.NoError(b, err)
+		assert.Equal(b, `"val3"`, val)
```

Comment:
You should benchmark only this code, initialization can be moved upper and reused.

Reply:
Also request context allocation can be reused:
```
reqCtx := &receivectx.Context{Body: body, Headers: header}
```